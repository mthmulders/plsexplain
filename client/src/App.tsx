import React from 'react';

const App = (): React.ReactElement => (<h1>Hello world</h1>);

export default App;
