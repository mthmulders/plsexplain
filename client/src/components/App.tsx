import React from 'react';
import NavigationBar from './NavigationBar';

const App = (): React.ReactElement => (
  <>
    <div className="container">
      <div className="row">
        <div className="col">
          <NavigationBar />
        </div>
      </div>
      <div className="row">
        <div className="col">
          <h1>Hello world</h1>
        </div>
      </div>
    </div>
  </>
);

export default App;
