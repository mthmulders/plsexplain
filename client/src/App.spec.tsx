import React from 'react';
import App from './App';
import renderer from 'react-test-renderer';

test('Renders succesfully', () => {
  const component = renderer.create(<App/>);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
