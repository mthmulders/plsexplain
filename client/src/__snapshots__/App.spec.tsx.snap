// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders succesfully 1`] = `
<h1>
  Hello world
</h1>
`;
